_posts/0000-01-02-Shubham-agrawal-hub.md

```diff
@@ -0,0 +1,5 @@
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
+Use the left arrow to go back!
```

Comment:
Ho the hell what is git i am moving around but unable to learn what exact i am trying to do